src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,335 @@
+
+  <include resource="io/knotx/logging/logback/defaults.xml"/>
+  <include resource="io/knotx/logging/logback/file-appender.xml"/>
+  <include resource="io/knotx/logging/logback/access.xml"/>
+
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
+      </Pattern>
+    </layout>
+  </appender>
+
+  <logger name="io.knotx" level="TRACE"/>
+  <logger name="io.vertx" level="DEBUG"/>
+  <logger name="com.codahale.metrics" level="TRACE"/>
+
+  <root level="TRACE">
+    <appender-ref ref="CONSOLE"/>
+  </root>
+</configuration>
+``` 
+
+## Operations configuration
+Now we have to configure `knotx/conf/routes/operations.conf`. To get more familiar with it please check out [this tutorial](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/).
+In our config we should have define two operations:
+- `healthcheck-operation`
+- `example-cacheable-api-operation`
+
+This operations are connected with `openapi.yaml`. As you remember, all endpoint definitions are connected with proper operations.
+According to it let's fill `knotx/conf/routes/operations.conf` with following:
+```hocon
+routingOperations = ${routingOperations} [
+  {
+    operationId = example-cacheable-api-operation
+    handlers = ${config.server.handlers.common.request} [
+      {
+        name = singleFragmentSupplier
+        config = {
+          type = json
+          configuration {
+            data-knotx-task = api-task
+          }
+        }
+      },
+      {
+        name = fragmentsHandler
+        config = { include required(classpath("routes/handlers/fragments-handler.conf")) }
+      },
+      {
+        name = fragmentsAssembler
+      }
+    ] ${config.server.handlers.common.response}
+  }
+  {
+    operationId = healthcheck-operation
+    handlers = [
+      {
+        name = healthcheck
+      }
+    ]
+  }
+]
+```
+
+Here we have defined all routing operations used in our tutorial, for us important is `example-cacheable-api-operation`.
+First thing we should focus on is `handlers` array, there we can specify which handlers should be used to process our 
+request. In our case, first handler to be called is `singleFragmentSupplier`. It's used to convert incoming http request
+into `knotx-fragment` and assign `data-knotx-task` to it, task definition will be provided soon. Next to be called is
+`fragmentsHandler`, here we have to attach additional configs with tasks specification. The last one is `fragmentsAssembler`
+Purpose of using it, is to combine all fragments into final response.
+
+## Handlers definition
+Last thing to do, is to define custom handler (Last thing about Knot.x configs, we have to create also Wiremock server). 
+As I mentioned above it will be `fragments-handler`.
+To do it let's remove from `knotx/conf/routes/handlers/` all handlers provided by default and create in this directory file
```

Comment:
if you remove all configs, the `healthcheck` will stop working

Reply:
in this directory provided is only one defaut handler, it will not affect healthcheck if directory content will be removed